README.md

```diff
@@ -55,19 +63,19 @@ services:
     container_name: decluttarr
     restart: always
     environment:
-      - TZ=Europe/Zurich
```

Comment:
I would leave TZ, since that is a docker standard. 
See my comment further down below